app/src/ui/diff/text-diff.tsx

```diff
@@ -773,7 +913,12 @@ export class TextDiff extends React.Component<ITextDiffProps, {}> {
       return
     }
 
-    const { file, hunks, readOnly } = this.props
+    // Discard right click events
```

Comment:
This comment read to me as  "ignore right click events". Might I suggest the following rewording?

```suggestion
    // We only care about the primary button here, secondary
    // button clicks are handled by `onContextMenu`
```